### PR TITLE
Inherit dataset value - dates

### PR DESCRIPTION
**Pull request by tilen1976**
- Add dates to existing methods 
- Remove stringify method for dates and change method for getting value i period fields after fix different date-formats
- Update tests with dates - parameterize tests
- Remove logging from get multilanguage metadata
- Fix indent error in doctest

### CODE REVIEW

File: src/datadoc/frontend/callbacks/variables.py [hunk 1]
```diff
@@ -244,6 +244,10 @@ def get_corresponding_identifier(
             return VariableIdentifiers.TEMPORALITY_TYPE
         case DatasetIdentifiers.DATA_SOURCE:
             return VariableIdentifiers.DATA_SOURCE
+        case DatasetIdentifiers.CONTAINS_DATA_FROM:
+            return VariableIdentifiers.CONTAINS_DATA_FROM
+        case DatasetIdentifiers.CONTAINS_DATA_UNTIL:
```
Comment (mmwinther) [inline src/datadoc/frontend/callbacks/variables.py:249]: We should implement this method as a dict, ref: https://docs.astral.sh/ruff/rules/too-many-branches/

File: tests/frontend/callbacks/test_variables_callbacks.py [hunk 2]
```diff
@@ -310,76 +310,131 @@ def test_populate_variables_workspace_filter_variables(
...
+        ),
+        (
+            "1988-11-03",
+            DatasetIdentifiers.CONTAINS_DATA_UNTIL,
+            VariableIdentifiers.CONTAINS_DATA_UNTIL,
+            "2008-01-01",
+        ),
+    ],
+)
+def test_variables_values_can_be_changed_after_update_from_dataset_value(
```
Comment (mmwinther) [inline tests/frontend/callbacks/test_variables_callbacks.py:399]: I'm not quite sure what this is testing, is there a risk that the variables values couldn't be updated?

File: tests/frontend/callbacks/test_variables_callbacks.py [hunk 3]
```diff
@@ -310,76 +310,131 @@ def test_populate_variables_workspace_filter_variables(
...
+            VariableIdentifiers.CONTAINS_DATA_FROM,
+        ),
+        (
+            "2021-08-10",
+            DatasetIdentifiers.CONTAINS_DATA_UNTIL,
+            VariableIdentifiers.CONTAINS_DATA_UNTIL,
+        ),
+    ],
+)
+def test_update_variables_values_from_dataset_values(
```
Comment (mmwinther) [inline tests/frontend/callbacks/test_variables_callbacks.py:342]:
```suggestion
def test_variables_values_inherit_dataset_values(
```

I think this is a clearer test name, do you agree?

**Review by mmwinther — CHANGES_REQUESTED** [review]
A couple of small comments. Fantastic to be rid of the `get_date_metadata_and_stringify` method though 😄

File: tests/frontend/callbacks/test_variables_callbacks.py (hunk 2 above)
Comment (tilen1976) [reply tests/frontend/callbacks/test_variables_callbacks.py:399]: It was a risk when developing and important to check that the inherited value was persisted in rest of the variables even after individual values were changed. But whether or not it is still important to test for this now that the code works I don't know...

**Review by mmwinther — APPROVED** [review]
Nice one 👍

**Pull request merged**